SwordPlugin.inc.php

```diff
@@ -286,5 +327,31 @@ public function getInstallSchemaFile() {
 	function getInstallEmailTemplatesFile() {
 		return ($this->getPluginPath() . '/emailTemplates.xml');
 	}
-}
 
+	/**
+	 * @see PKPPlugin::getInstallEmailTemplatesFile()
+	 */
+	function getInstallEmailTemplatesFile() {
+		return ($this->getPluginPath() . '/emailTemplates.xml');
+	}
+
+	/**
+	 * @see PKPPlugin::getInstallEmailTemplateDataFile()
+	 */
+	function getInstallEmailTemplateDataFile() {
+		return ($this->getPluginPath() . '/locale/{$installedLocale}/emailTemplates.xml');
+	}
+
+	/**
+	 * Make sure the installedLocales are set when
+	 * Installer->preInstall is not run, i.e., when
+	 * using the install plugin CLI tool
+	 */
+	function installEmailTemplateData($hookName, $args) {
```

Comment:
Is this necessary to get the email templates installed? If so, there may be an underlying bug that needs fixing.

Reply:
(Removed)